dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/viewtools/DotTemplateToolTest.java

```diff
@@ -0,0 +1,81 @@
+package com.dotcms.rendering.velocity.viewtools;
+
+import com.dotcms.datagen.ContainerDataGen;
+import com.dotcms.datagen.TemplateDataGen;
+import com.dotcms.datagen.TemplateLayoutDataGen;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.portlets.containers.model.Container;
+import com.dotmarketing.portlets.templates.design.bean.ContainerUUID;
+import com.dotmarketing.portlets.templates.design.bean.TemplateLayout;
+import com.dotmarketing.portlets.templates.model.Template;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+
+public class DotTemplateToolTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception{
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6034685)